settings/common.py

```diff
@@ -172,6 +177,20 @@
 
 # -- Auth
 LOGIN_URL = reverse_lazy("login")
+
+if DEBUG is False:
```

Comment:
Perhaps a bit more pythonic:
```suggestion
if not DEBUG:
```